### PR TITLE
Updates for ceph benchmarking

### PR DESCRIPTION
**Pull request by thedac**
Set OSD devices
Remove keystone and use auto unseal for vault

### CODE REVIEW

File: tests/ceph-benchmarking/tests/bundles/bionic-nautilus.yaml [hunk 1]
```diff
@@ -4,15 +4,11 @@ applications:
     num_units: 1
     series: bionic
     charm: vault
+    options:
+      totally-unsecure-auto-unlock: True
```
Comment (fnordahl) [inline tests/ceph-benchmarking/tests/bundles/bionic-nautilus.yaml:8]: Would it not be possible to remove keystone and use the `zaza.openstack.charm_tests.vault.setup.auto_initialize_no_validation` ([example](https://opendev.org/x/charm-ovn-central/src/branch/master/src/tests/tests.yaml#L21)) instead? I am a bit skeptic to us using the `totally-unsecure-auto-unlock` configuration option and would hope we could remove it one day.

**Review by fnordahl — CHANGES_REQUESTED** [review]
See in-line.

File: tests/ceph-benchmarking/tests/bundles/bionic-nautilus.yaml (hunk 1 above)
Comment (thedac) [reply tests/ceph-benchmarking/tests/bundles/bionic-nautilus.yaml:8]: I'll test that.

**Review by fnordahl — APPROVED** [review]
LGTM

**Pull request merged**